lib/view_model/wallet_address_list/wallet_address_list_view_model.dart

```diff
@@ -603,7 +565,8 @@ abstract class WalletAddressListViewModelBase
         WalletType.haven,
         WalletType.bitcoinCash,
         WalletType.bitcoin,
-        WalletType.litecoin
+        WalletType.litecoin,
+        WalletType.decred
```

Comment:
Why are we doing this again? Seems like we can simply say false to `hasAddressList`...

Reply:
When the wallet is not synced (disconnected), the receive page doesn't show an address that can be used but the list of addresses shows.

Reply:
idk, the guy asked me to implement them, it will show used still.

I guess it makes sense to show used ones and labeling them for some people.

Reply:
I want to see how this feature works for testnet btc but getting test coins has been a challenge. The faucet I used sent a tx with a fee that is quite lower than most txs in the mempool, so no idea when that will get mined.

Reply:
bitcoin testnet coins? yeah...